index.html

```diff
@@ -289,11 +131,11 @@ <h6 class="mdc-list-group__subheader">Friends of CD</h6>
   <header class="mdc-top-app-bar mdc-top-app-bar--fixed">
     <div class="mdc-top-app-bar__row">
       <section class="mdc-top-app-bar__section mdc-top-app-bar__section--align-start">
-        <a class="material-icons mdc-top-app-bar__navigation-icon">menu</a>
-        <span class="mdc-top-app-bar__title">Community Projects</span>
+        <button class="material-icons mdc-top-app-bar__navigation-icon mdc-icon-button">menu</button>
+        <span class="mdc-top-app-bar__title">CD Community</span>
```

Comment:
```suggestion
        <span class="mdc-top-app-bar__title">Cyber Discovery Community</span>
```

Reply:
@thebeanogamer I am happy to make this change just like the other changes, but please note that doing this means that for people on mobile devices with < 5.5 inch screens, the text will be truncated and they will see some dots.

![Demo2](https://user-images.githubusercontent.com/28938427/61568460-43667c00-aa7b-11e9-81ee-432c09003e22.png)

Reply:
That's... less than ideal

Reply:
Ok, I'll do it nonetheless! :+1: